config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -12,9 +12,11 @@ dashboard_groups:
     - sig-node-node-feature-discovery
     - sig-node-node-problem-detector
     - sig-node-seccomp-operator
+    - sig-node-blocking
 
 dashboards:
 - name: sig-node-cadvisor
+- name: sig-node-blocking # tabs included in this group are defined in the jobs that are included
```

Comment:
it would be so nice to have a way to add a description to the tab to show in Summary view. Do you know if this is supported?

Reply:
https://github.com/GoogleCloudPlatform/testgrid/blob/96bf6f1712d54474d69f483ccbe49fa4f9ae0204/pb/config/config.proto#L539-L540

Maybe? 

I've not encountered any existing things that look like that. Have you seen one? It would be easy to find  with an example?